recipes/sagemath-db-combinatorial-designs/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+mkdir -p "$PREFIX/share/combinatorial_designs"
+mv MOLS_table.txt "$PREFIX/share/combinatorial_designs"
```

Comment:
Seems like a lot of boilerplate to package a data table :smile: 
Nothing against it but I believe there are more light-weight ways to do so :wink: